### DISCUSSION TITLE
Whiskey dosen't want to install on mac mini 2023

### DISCUSSION THREAD

**Discussion by ObviosLyRed444** [Whisky-App/Whisky#1071] [Q&A]
Help, whenever I try to install whisky on my mac mini sonoma M2 2023 because every time when i press next it says(on the install page) that zero kb of zero kb. I thought I might try a different way to install it a different way so i installed whiskeywine which i did through homebrew. It said it was successful but when i open it( the one on through homebrew) it still gives out the same thing and nothing happens. What do I do or is there some advise on how to get it to work? Am I missing something?
![Screenshot 2024-07-30 at 11 57 40 AM](https://github.com/user-attachments/assets/1ea32136-626f-4194-89b9-f9424956a4a3)
![Screenshot 2024-07-30 at 12 01 39 PM](https://github.com/user-attachments/assets/97575fe5-2981-4804-8c6b-d0614f0ade92)
![Screenshot 2024-07-30 at 12 05 18 PM](https://github.com/user-attachments/assets/32b27c8f-515d-4b9b-be18-f19df3035115)I'll also include my mac's specs

**Comment by lazywhite** [comment]
maybe network error, try system proxy
